src/components/Play/PlayContainer/PlayContainer.js

```diff
@@ -0,0 +1,20 @@
+import Player from "src/components/Play/Player/Player.js";
+import Comments from "src/components/Play/Comments/Comments.js";
+import "./PlayContainer.css";
+import contentsData from "src/mock/contents.json";
+
+export default function PlayContainer(props) {
+  var id = props.id;
+  let content = contentsData[id];
```

Comment:
```suggestion
  const content = contentsData[id];
```

요것도 런타임에서 값이 바뀌지 않는 값이라 굳이 let을 쓸 거 없이 const를 써도 될 것 같네용